config.md

```diff
@@ -87,8 +87,8 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 
 * **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process. Defaults to false.
 * **`cwd`** (string, required) is the working directory that will be set for the executable. This value MUST be an absolute path.
-* **`env`** (array of strings, optional) contains a list of variables that will be set in the process's environment prior to execution. Elements in the array are specified as Strings in the form "KEY=value". The left hand side must consist solely of letters, digits, and underscores `_` as outlined in [IEEE Std 1003.1-2001](http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap08.html).
-* **`args`** (string, required) executable to launch and any flags as an array. The executable is the first element and must be available at the given path inside of the rootfs. If the executable path is not an absolute path then the search $PATH is interpreted to find the executable.
+* **`env`** (array of strings, optional) contains a list of variables that will be set in the process's environment prior to execution. Elements in the array are specified as Strings in the form "KEY=value". The left hand side MUST consist solely of letters, digits, and underscores `_` as outlined in [IEEE Std 1003.1-2001](http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap08.html).
+* **`args`** (string, required) executable to launch and any flags as an array. The executable is the first element and MUST be available at the given path inside of the rootfs. If the executable path is not an absolute path then the search $PATH is interpreted to find the executable.
```

Comment:
This is awkward.  The “… first element…” sentence is a bundle requirement, but I really don't care if we require that or not.  Bundles that don't have an executable at that path (in the container's mount namespace after all the mounts, etc., etc. are setup?) will fail.  But *checking* for that ahead of time, without actually launching the container, seems difficult and/or unstable.  I'd rather reword this to cite [`execv(3)`][1] informatively, but allow bundle validators to ignore it.

The “… `$PATH`…” sentence is a runtime requirement, and probably needs a “MUST” and a link to something more explicit (like [`execv(3)`][1]).  Do we need to distinguish between bundle and runtime requirements in this file (see also #404)?

And if both of those points are too involved for this PR and I should spin them out into follow-up issues/PRs, that's fine with me.

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/functions/execv.html